### PR TITLE
Tests/Add testID to AnimatedScanner

### PR DESCRIPTION
**Pull request by audrius-savickas**
## Description
Added a testID to `AnimatedScanner` to allow easier testing

## Changelog
AnimatedScanner - new `testID` prop - added `testID` prop to use for testing the component

### CODE REVIEW

**Review by Inbal-Tish — APPROVED** [review]

**Comment by Inbal-Tish** [comment]
@audrius-savickas Please fix lint error

File: src/components/animatedScanner/index.js [hunk 1]
```diff
@@ -45,7 +45,11 @@ export default class AnimatedScanner extends BaseComponent {
     /**
      * the container style
      */
-    containerStyle: ViewPropTypes.style
+    containerStyle: ViewPropTypes.style,
+    /**
+     * Used as a testing identifier
+     */
+    testID: PropTypes.string,
```
Comment (Inbal-Tish) [inline src/components/animatedScanner/index.js:52]:
```suggestion
    testID: PropTypes.string
```

Comment (Inbal-Tish) [reply src/components/animatedScanner/index.js:52]: This is the lint error fix

Comment (audrius-savickas) [reply src/components/animatedScanner/index.js]: thanks!

**Pull request merged**